src/openrct2-ui/windows/Options.cpp

```diff
@@ -894,27 +897,6 @@ namespace OpenRCT2::Ui::Windows
                 disabled_widgets &= ~(1uLL << WIDX_RESOLUTION_LABEL);
             }
 
-            // Disable Steam Overlay checkbox when using software or OpenGL rendering.
-            if (Config::Get().general.DrawingEngine == DrawingEngine::Software
-                || Config::Get().general.DrawingEngine == DrawingEngine::OpenGL)
-            {
-                disabled_widgets |= (1uLL << WIDX_STEAM_OVERLAY_PAUSE);
-            }
-            else
-            {
-                disabled_widgets &= ~(1uLL << WIDX_STEAM_OVERLAY_PAUSE);
-            }
-
-            // Disable changing VSync for Software engine, as we can't control its use of VSync
-            if (Config::Get().general.DrawingEngine == DrawingEngine::Software)
-            {
-                disabled_widgets |= (1uLL << WIDX_USE_VSYNC_CHECKBOX);
-            }
-            else
-            {
-                disabled_widgets &= ~(1uLL << WIDX_USE_VSYNC_CHECKBOX);
-            }
-
```

Comment:
Why was this removed altogether? Surely it should still disable it for OpenGL unless I missed something?

Reply:
I should have removed the entire thing, its not functional for any engine and unless we use the Steam API there is no good way of telling also.

Reply:
I don't personally use Steam, but if it's broken, I'd be interested to learn when it broke. Do you think something changed on Steam's side? The way the overlay is detected looks a little rudimentary, so it's surprising it's lasted as long as it did, I suppose.

Reply:
I don't know when it broke there is also no way to revert Steam to an old version, but yes sampling pixels before and after a frame is quite a special way to see if something renders into the frame buffer, Windows also has this new API which adds proper overlays which will not affect the frame buffer and they might be using that now, I can't say for sure, all I can say is that I tested it with a couple releases and it doesn't work.

Reply:
That's unfortunate. We _should_ consider removing it, then, or at least file an issue to discuss what to do.

Reply:
#24326